kura/org.eclipse.kura.container.orchestration.provider/pom.xml

```diff
@@ -37,132 +37,115 @@
-                                    <artifactId>httpcore5</artifactId>
-                                    <version>5.3.4</version>
-                                </artifactItem>
-                                <artifactItem>
-                                    <groupId>net.java.dev.jna</groupId>
-                                    <artifactId>jna</artifactId>
-                                    <version>5.17.0</version>
-                                </artifactItem>
-                                <artifactItem>
-                                    <groupId>commons-codec</groupId>
-                                    <artifactId>commons-codec</artifactId>
-                                    <version>1.18.0</version>
-                                </artifactItem>
-                            </artifactItems>
-                        </configuration>
-                        <goals>
-                            <goal>copy</goal>
-                        </goals>
-                    </execution>
-                </executions>
-            </plugin>
             <plugin>
                 <artifactId>maven-clean-plugin</artifactId>
-                <version>3.1.0</version>
                 <configuration>
+                    <verbose>true</verbose>
                     <filesets>
                         <fileset>
-                            <directory>lib</directory>
+                            <directory>lib/</directory>
+                            <followSymlinks>false</followSymlinks>
                         </fileset>
                     </filesets>
                 </configuration>
             </plugin>
             <plugin>
-                <groupId>de.dentrassi.maven</groupId>
-                <artifactId>osgi-dp</artifactId>
-                <version>0.4.1</version>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.8.1</version>
                 <executions>
                     <execution>
+                        <id>copy-dependencies</id>
+                        <phase>generate-sources</phase>
                         <goals>
-                            <goal>build</goal>
+                            <goal>copy-dependencies</goal>
                         </goals>
+                        <configuration>
+                            <outputDirectory>${project.basedir}/lib</outputDirectory>
+                            <includeArtifactIds>
+                                docker-java, docker-java-core, docker-java-api,
+                                docker-java-transport, docker-java-transport-httpclient5,
+                                httpclient5, httpcore5, httpcore5-h2, jna, commons-codec,
+                                jackson-databind, jackson-core, jackson-annotations,
+                                commons-compress
+                            </includeArtifactIds>
+                            <excludeTransitive>true</excludeTransitive>
```

Comment:
Since we are specifying the `includeArtifactIds`, we don't need `<excludeTransitive>true</excludeTransitive>` anymore